num_joints: 7
control_gain: 0.01
control_gain_pose: 0.0075
max_joint_vel: 0.3
scale_translation: 1.0
scale_rotation: 1.0